### PR TITLE
Avoid big numbers

### PR DESCRIPTION
**Pull request by 0xbala-k**
Use sdktypes.Dec

### CODE REVIEW

**Comment by pitasi** [comment]
I think you can use `NewDecFromStr` instead of `NewIntFromStr`. This func returns a `sdktypes.Dec` (dec stands for decimal), and then you can do `.Quo` between them.

You basically can have:
```
bondedTokens, err := sdktypes.NewDecFromStr(stakingPoolData.Pool.BondedTokens)

supply := coin.Amount.ToDec()

inflation, err := sdktypes.NewDecFromStr(inflationData.Inflation)

apr := inf.Quo(bond).Quo(supp).MulInt64(100) // check this math 'cause I wrote this quickly
return apr.String()
```

**Review by pitasi — CHANGES_REQUESTED** [review]
see prev comment

File: api/chains/chains.go [hunk 1]
```diff
@@ -1457,7 +1461,7 @@ func getAPR(c *gin.Context) stringcache.HandlerFunc {
 		}
 
 		// calculate staking APR
-		apr := (inflation * 100) / (float64(bondedTokens) / float64(supply.Uint64()))
+		apr := inflation.Quo(bondedTokens.Quo(supply)).MulInt64(100)
 		return fmt.Sprintf("%f", apr), nil
```
Comment (pitasi) [inline api/chains/chains.go]:
```suggestion
		return apr.String(), nil
```

**Review by akhilkumarpilli — APPROVED** [review]

File: api/chains/chains.go [hunk 2]
```diff
@@ -1457,7 +1461,7 @@ func getAPR(c *gin.Context) stringcache.HandlerFunc {
 		}
```
Comment (pitasi) [inline api/chains/chains.go:1458]: some lines above this one it says `float64` but should say `sdktypes.Dec` for consistency instead

**Review by pitasi — CHANGES_REQUESTED** [review]

**Review by pitasi — APPROVED** [review]

**Pull request merged**